config/settings/base.py

```diff
@@ -619,6 +619,12 @@ def set_logging(level=None):
 PROJECTROLES_SEARCH_OMIT_APPS = env.list(
     'PROJECTROLES_SEARCH_OMIT_APPS', None, []
 )
+PROJECTROLES_TARGET_SYNC_ENABLE = env.bool(
```

Comment:
Make sure to document these new Django settings in `app_projectroles_settings.rst` under "Optional Projectroles Settings" :)

Reply:
It looks like `PROJECTROLES_TARGET_SYNC_INTERVAL` is still missing from the doc.